src/utils/file-extension.ts

```diff
@@ -0,0 +1,24 @@
+import { extname } from 'path';
+import { TextDocumentIdentifier } from 'vscode-languageserver';
+import { uriToFilePath } from 'vscode-languageserver/lib/files';
+
+export function getExtension(textDocument: TextDocumentIdentifier): string | null {
+  const filePath = uriToFilePath(textDocument.uri);
+  const ext = filePath ? extname(filePath) : '';
+
+  switch (ext) {
+    case '.hbs':
+    case '.handlebars':
+      return '.hbs';
+    case '.js':
```

Comment:
That's true I started with a Enum as return type... Anyway, would you prefer, to get rid of the switch-case:

```
if (ext === .handlebars') {
  return '.hbs';
}
return ext;
```